config/app_config.cfg

```diff
@@ -9,6 +9,7 @@ DPNAME = ngic-skx2
 MCC = 000
 MNC = 000
 TAC = 0
+IMSI = 208014567891200
```

Comment:
I think we should do it IMSI range based instead of doing single IMSI based.

Reply:
Selecting DP is typically not ideal as you select the CP and then let data it collects to select the DP.  

There are a few ways to select CP and add data that helps select the DP.
1. Adding the dedicated core information to the HSS for the UE which is then used in DNS lookups to get the correct CP/DP (this would require an update to the MME to look at the information when provided by the HSS).
2  Configuration Lookup of the DNS.

MME performing special actions by IMSI range is very common and can leverage one or both methods above.  I would definitely support it and think it is important for the MME for many other activities.

Depending upon the approach one can also perform an eNodeB DNS based lookup for dataplane or default to the TAC method which I think is already supported.  In that case the DNS data and SGW-C / SAEGW-C configurations are key.  I believe we already support that but maybe need confirmation as to which branches support it.

Reply:
@lbertz02  yes, good approaches, I will try to implement it in the future, but as the current CP code make it easy to implement this feature, I preferred to add it here,

Reply:
> I think we should do it IMSI range based instead of doing single IMSI based.

@thakurajayL could you check the modification, I have commited the changes to support of IMSI range,
Thank you